third_party/libpg_query/grammar/statements/select.y

```diff
@@ -3080,7 +3080,7 @@ window_specification: '(' opt_existing_window_name opt_partition_clause
 		;
 
 /*
- * If we see PARTITION, RANGE, or ROWS as the first token after the '('
+ * If we see PARTITION, RANGE, ROWS or GROUPS as the first token after the '('
```

Comment:
> Does this also mean that windows cannot be called partition, range, and rows?

Yes. I didn't write tests for them because my changes to the parser only affected `groups`, but they could be added for the sake of completeness.